src/transformers/models/idefics/clip.py

```diff
@@ -197,8 +111,8 @@ def forward(self, pixel_values: torch.FloatTensor) -> torch.Tensor:
         return embeddings
 
 
-# copied from transformers.models.clip.modeling_clip.CLIPAttention
-class CLIPAttention(nn.Module):
+# Copied from transformers.models.clip.modeling_clip.CLIPAttention with CLIP->IdeficsVision
```

Comment:
let's use s/Copied/Adapted/, since Copied looks too much like copied and then the CI checker might not like it since it's likely someone is going to "fix" it to lowercase.

```suggestion
# Adaped from transformers.models.clip.modeling_clip.CLIPAttention with CLIP->IdeficsVision
```

Reply:
same for other classes